s3_to_redshift/s3_to_redshift.py

```diff
@@ -436,6 +478,33 @@ def sortobjects_last_modified(o):
     # map the dataframe column names to match the columns from the configuation
     df.columns = columns
 
+
+    # Check for empty file that has zero data rows
+    if len(df.index) == 0:
+        logger.info('%s is empty and contains zero data rows, keying to badfile and no further processing.',
```

Comment:
I believe the file can contain a header, so this log message is wrong.

Instead I suggest:
```
'%s is contains zero data rows, keying to badfile and no further processing.'
```